docs/guidance/build/merging.md

```diff
@@ -12,50 +12,38 @@ The following examples show how updates and deletions are reflected in compiled
 
 ## Example 1: Updates
 
-A public procurement agency publishes a release to announce an opportunity on January 1, 2016 in which the estimated value of the procurement is $1,000. On January 31, it publishes a release to expand the description of the procurement. On February 5, it publishes a release to amend the opportunity, in which the estimated value is increased to $2,000.
+A government agency in Ghana publishes a release to announce an opportunity on October 21, 2020 with an estimated value of 13,000 GHS. On November 5, 2020 they publish an update containing the details of 2 parties that have submitted bids and the publication date of the bidding documents.
 
-The agency decides to award the opportunity to two of the bidders. On March 1, the agency publishes a release to announce that Company A is awarded a contract of $750. On March 3, the agency publishes a release to announce that Company B is awarded a contract of $750.
+The agency then decides to cancel the tender. On December 19, 2020 it publishes a third release announcing that the tender has been cancelled.
```

Comment:
Remove the first sentence because it's likely the buyer/procuring entity that decides to cancel the tender rather than the publisher.

```suggestion
On December 19, 2020 it publishes a third release announcing that the tender is cancelled.
```